js/google_ima.js

```diff
@@ -299,6 +301,11 @@ require("../html5-common/js/utils/utils.js");
         {
           _IMA_SDK_tryInitAdContainer();
         }
+        else if (!_IMAAdsLoader)
```

Comment:
Yep, the AdsLoader is dependent on the Ad Container. The current flow is the create the Ads Loader after the Ad Container is created.